.gitignore

```diff
@@ -56,6 +56,7 @@ client_body_temp
 utils/lj-releng
 .idea/
 *.iml
+dashboard/
```

Comment:
dashboard will be integrated later

Reply:
we can remove it after integrated the dashboard.